integration-cli/docker_cli_logs_test.go

```diff
@@ -178,6 +178,41 @@ func (s *DockerSuite) TestLogsSince(c *check.C) {
 	}
 }
 
+func (s *DockerSuite) TestLogsUntil(c *check.C) {
```

Comment:
@jamiehannaford the old CLI integration tests are still here to watch for regressions, but there is a tentative plan to move them out to another repo, the details of which haven't been fully hammered out. 

The functional effect is just that you should rewrite your tests to target the API instead of the CLI. Sorry about the trouble.